test/sql-tap/CMakeLists.txt

```diff
@@ -9,3 +9,104 @@ build_module(sql_uuid sql_uuid.c)
 target_link_libraries(sql_uuid msgpuck core)
 build_module(decimal decimal.c)
 target_link_libraries(decimal msgpuck)
+
+make_lua_path(LUA_PATH
+  PATHS
+    ${CMAKE_CURRENT_SOURCE_DIR}/lua/?.lua
+    ${PROJECT_SOURCE_DIR}/test/sql/lua/?.lua
+    ${PROJECT_SOURCE_DIR}/test/box/lua/?.lua
+)
+
+make_lua_path(LUA_CPATH
+  PATHS
+  ${CMAKE_CURRENT_BINARY_DIR}/?.so
+)
+
+set(TEST_SUITE_NAME "sql-tap")
+set(TARANTOOL_BIN $<TARGET_FILE:tarantool>)
+
+message(STATUS "Add test suite ${TEST_SUITE_NAME}")
+
+# XXX: The call produces both test and target <sql-tap-deps>
+# as a side effect.
+_add_test_suite_target(${TEST_SUITE_NAME}
+  LABELS ${TEST_SUITE_NAME}
+  DEPENDS gh-5938-wrong-string-length gh-6024-funcs-return-bin gh-2579-custom-aggregate sql_uuid decimal
```

Comment:
Please add the comment about LuaC libraries.
Also, I found that <gh-5938-wrong-string-lengt> hthe can't be run as a single test:

```sh
ctest -V -R gh-5938-wrong-string-length
...
1456: Test timeout computed to be: 1500
1456: 2024-09-27 13:47:35.892 [13785] main F> fatal error, exiting the event loop
1456: TAP version 13
1456: 1..2
1/1 Test #1456: test/sql-tap/gh-5938-wrong-string-length.test.lua ...***Failed    0.42 sec
0% tests passed, 1 tests failed out of 1
```

Reply:
Now it is succeeded:

```
test 1461
    Start 1461: test/sql-tap/gh-5938-wrong-string-length.test.lua

1461: Test command: /bin/bash "-c" "/bin/rm -f /home/sergeyb/sources/MRG/tarantool/build/test/sql-tap/gh-5938-wrong-string-length.test.lua-workdir/* && /home/sergeyb/sources/MRG/tarantool/build/src/tarantool /home/sergeyb/sources/MRG/tarantool/test/sql-tap/gh-5938-wrong-string-length.test.lua"
1461: Working Directory: /home/sergeyb/sources/MRG/tarantool/test/sql-tap
1461: Environment variables: 
1461:  LUA_PATH=/home/sergeyb/sources/MRG/tarantool/test/sql-tap/lua/?.lua;/home/sergeyb/sources/MRG/tarantool/test/sql/lua/?.lua;/home/sergeyb/sources/MRG/tarantool/test/box/lua/?.lua;;
1461:  LUA_CPATH=/home/sergeyb/sources/MRG/tarantool/build/test/sql-tap/?.so;;
1461:  BUILDDIR=/home/sergeyb/sources/MRG/tarantool/build/test/sql-tap
1461:  TT_WORK_DIR=/home/sergeyb/sources/MRG/tarantool/build/test/sql-tap/gh-5938-wrong-string-length.test.lua-workdir
1461: Test timeout computed to be: 1500
1461: TAP version 13
1461: 1..2
1461: ok - gh-5938-1
1461: ok - gh-5938-2
1/1 Test #1461: test/sql-tap/gh-5938-wrong-string-length.test.lua ...   Passed    0.16 sec

The following tests passed:
        test/sql-tap/gh-5938-wrong-string-length.test.lua

100% tests passed, 0 tests failed out of 1

Label Time Summary:
integration    =   0.16 sec*proc (1 test)
sql-tap        =   0.16 sec*proc (1 test)

Total Test time (real) =   0.35 sec
```

Could you check once again?

Reply:
Yes, works like charm. Thanks!